pyglet/image/codecs/pil.py

```diff
@@ -135,7 +135,10 @@ def encode(self, image, file, filename):
 
         # fromstring is deprecated, replaced by frombytes in Pillow (PIL fork)
         # (1.1.7) PIL still uses it
-        image_from_fn = getattr(Image, "frombytes", getattr(Image, "fromstring"))
+        try:
```

Comment:
no, here this is wrong on two accounts
 - needs to be 'frombytes' and 'fromstring', not 'tobytes' and 'tostrings'
 - it should be Image , not image 
see #297 if any doubt

Reply:
Ooops really sorry, that was me trying to fix stuff quickly over the weekend. Thanks for looking at this, you are right about your changes.